### PR TITLE
Replace render_to_response with render for better Django 1.8 compatibility.

### PR DESCRIPTION
**Pull request by mythmon**
This makes Badger work with new style non-default templating engines. (like django-jinja)

This makes Sumo much happier.

@willkg r?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,14 @@ python:
 env:
     - DB=mysql DJANGO_VERSION=1.4
     - DB=mysql DJANGO_VERSION=1.5
+    - DB=mysql DJANGO_VERSION=1.6
+    - DB=mysql DJANGO_VERSION=1.7
```
Comment (willkg) [inline .travis.yml]: It doesn't matter much, but I'd just ditch everything other than 1.8.

File: tox.ini [hunk 2]
```diff
@@ -7,40 +7,47 @@
...
            pip install -r requirements/dev.txt
-           {envpython} manage.py test -v2 badger
+           {envpython} manage.py test badger
 
 [testenv:py27_dj17]
 basepython = python2.7
 commands = pip install "Django>=1.7,<1.7.99"
            pip install -r requirements/dev.txt
-           {envpython} manage.py test -v2 badger
+           {envpython} manage.py test badger
```
Comment (willkg) [inline tox.ini]: I'd ditch everything except the 1.8 runs here, too.

**Comment by willkg** [comment]
It looks fine. I mentioned ditching all but Django 1.8 testing. But it occurs to me we should do that in a separate PR. I'll do that after this lands.

**Comment by willkg** [comment]
If Travis is cool, r+!

**Comment by mythmon** [comment]
I screwed up the Travis exclusions for Django version vs Python 2.6. I fixed that now, and it should pass Travis soon

I didn't really care about removing the old versions of Django/Python, since the tests passed easily with them.

**Pull request merged**